reconcile/glitchtip/integration.py

```diff
@@ -153,10 +153,18 @@ def run(dry_run: bool, instance: Optional[str] = None) -> None:
             read_timeout=read_timeout,
             max_retries=max_retries,
         )
+        automation_user_email = secret_reader.read(
+            {
+                "path": glitchtip_instance.automation_user_email.path,
+                "field": glitchtip_instance.automation_user_email.field,
+                "format": glitchtip_instance.automation_user_email.q_format,
+                "version": glitchtip_instance.automation_user_email.version,
+            }
+        )
```

Comment:
Why not use `secret_reader.read_secret(glitchtip_instance.automation_user_email)`?

Reply:
Copy & paste :( Let me change it. Thx